scripts/lib/on_alert.js

```diff
@@ -100,17 +113,22 @@ function localizations() {
 
 function isPrivacyAlert(alert) {
 
-  var ans, exp, txt;
+  var expressions = localizations();
+
+  var title = findAlertTitle(alert);
 
-  var exps = localizations();
+  // When debugging or trying to capture the regexes for a new
+  // localization, uncomment these lines.
+  // var buttonNames = findAlertButtonNames(alert);
+  // Log.output({"output":"alert: " + title + "," + buttonNames}, true);
```

Comment:
@krukow 

I want to remove the logging.  It is very slow and delays the dismissal of the alert.

What do you think?

I added this text the Permissions README:

https://github.com/calabash/Permissions/blob/master/README.md#use-permissions-to-add-new-localization